### PR TITLE
Improve thumbnail generation

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by DunklesArchipel** [comment]
~~Looks ace.~~

~~I have a small glitch in Chrome, where the non-highlighted figure is blurry (with Chrome zoom factor 100%). This is not the case in Firefox.~~

~~The blur almost disappears for zoom 75% and inverts for smaller zoom factors.~~

sorry, that was supposed to show up here: https://github.com/echemdb/website/pull/159

File: echemdb/cv/entry.py [hunk 1]
```diff
@@ -370,7 +370,7 @@ def _normalize_field_name(self, field_name):
             return self._normalize_field_name("I")
         raise ValueError(f"No axis named '{field_name}' found.")
 
-    def thumbnail(self, width=2, height=1, **kwds):
+    def thumbnail(self, width=96, height=72, **kwds):
```
Comment (DunklesArchipel) [inline echemdb/cv/entry.py]: don't you want to specify `dpi` here?

Comment (DunklesArchipel) [reply echemdb/cv/entry.py]:
```suggestion
    def thumbnail(self, width=96, height=72, dpi=72, **kwds):
```

File: echemdb/cv/entry.py [hunk 2]
```diff
@@ -394,7 +394,12 @@ def thumbnail(self, width=2, height=1, **kwds):
 
         import matplotlib.pyplot
 
-        fig, axis = matplotlib.pyplot.subplots(1, 1, figsize=[width, height])
+        # A reasonable DPI setting that should work for most screens.
+        dpi = 72
+
```
Comment (DunklesArchipel) [inline echemdb/cv/entry.py]:
```suggestion
        # A reasonable DPI setting that should work for most screens is the default value of 72.
```

**Review by DunklesArchipel — APPROVED** [review]

**Comment by saraedum** [comment]
@DunklesArchipel thanks for the review. The DPI should be configurable of course.

**Pull request merged**